diff --git a/quartz/components/pages/TagContent.tsx b/quartz/components/pages/TagContent.tsx
index a1df6e1..739da9d 100644
--- a/quartz/components/pages/TagContent.tsx
+++ b/quartz/components/pages/TagContent.tsx
@@ -12,10 +12,14 @@ import { concatenateResources } from "../../util/resources"
 interface TagContentOptions {
   sort?: SortFn
   numPages: number
+  hideTags?: string[]
+  minEntries?: number
 }
 
 const defaultOptions: TagContentOptions = {
   numPages: 10,
+  hideTags: [],
+  minEntries: 1,
 }
 
 export default ((opts?: Partial<TagContentOptions>) => {
@@ -47,7 +51,10 @@ export default ((opts?: Partial<TagContentOptions>) => {
         ...new Set(
           allFiles.flatMap((data) => data.frontmatter?.tags ?? []).flatMap(getAllSegmentPrefixes),
         ),
-      ].sort((a, b) => a.localeCompare(b))
+      ]
+        .filter((t) => !(options.hideTags ?? []).includes(t))
+        .filter((t) => allPagesWithTag(t).length >= (options.minEntries ?? 1))
+        .sort((a, b) => a.localeCompare(b))
       const tagItemMap: Map<string, QuartzPluginData[]> = new Map()
       for (const tag of tags) {
         tagItemMap.set(tag, allPagesWithTag(tag))
@@ -78,29 +85,10 @@ export default ((opts?: Partial<TagContentOptions>) => {
               const href = resolveRelative(fileData.slug!, tagListingPage)
 
               return (
-                <div>
-                  <h2>
-                    <a class="internal tag-link" href={href}>
-                      {tag}
-                    </a>
-                  </h2>
-                  {content && <p>{content}</p>}
-                  <div class="page-listing">
-                    <p>
-                      {i18n(cfg.locale).pages.tagContent.itemsUnderTag({ count: pages.length })}
-                      {pages.length > options.numPages && (
-                        <>
-                          {" "}
-                          <span>
-                            {i18n(cfg.locale).pages.tagContent.showingFirst({
-                              count: options.numPages,
-                            })}
-                          </span>
-                        </>
-                      )}
-                    </p>
-                    <PageList limit={options.numPages} {...listProps} sort={options?.sort} />
-                  </div>
+                <div class="tag-index-entry">
+                  <a class="internal tag-link" href={href}>
+                    {tag} <span class="tag-count">({pages.length})</span>
+                  </a>
                 </div>
               )
             })}
