diff --git a/quartz/components/Head.tsx b/quartz/components/Head.tsx
index 23183ca..4dcc431 100644
--- a/quartz/components/Head.tsx
+++ b/quartz/components/Head.tsx
@@ -52,8 +52,10 @@ export default (() => {
         )}
         <link rel="preconnect" href="https://cdnjs.cloudflare.com" crossOrigin="anonymous" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="robots" content="noindex, nofollow" />
 
         <meta name="og:site_name" content={cfg.pageTitle}></meta>
+        <meta property="fb:app_id" content="1885135262369297" />
         <meta property="og:title" content={title} />
         <meta property="og:type" content="website" />
         <meta name="twitter:card" content="summary_large_image" />
