diff --git a/quartz/plugins/emitters/ogImage.tsx b/quartz/plugins/emitters/ogImage.tsx
index 813d934..11f89d3 100644
--- a/quartz/plugins/emitters/ogImage.tsx
+++ b/quartz/plugins/emitters/ogImage.tsx
@@ -62,7 +62,7 @@ async function generateSocialImage(
     },
   })
 
-  return sharp(Buffer.from(svg)).webp({ quality: 40 })
+  return sharp(Buffer.from(svg)).png({ quality: 90 })
 }
 
 async function processOgImage(
@@ -96,7 +96,7 @@ async function processOgImage(
     ctx,
     content: stream,
     slug: `${slug}-og-image` as FullSlug,
-    ext: ".webp",
+    ext: ".png",
   })
 }
 
@@ -154,24 +154,24 @@ export const CustomOgImages: QuartzEmitterPlugin<Partial<SocialImageOptions>> =
             }
 
             const generatedOgImagePath = isRealFile
-              ? `https://${baseUrl}/${pageData.slug!}-og-image.webp`
+              ? `https://${baseUrl}/${pageData.slug!}-og-image.png`
               : undefined
             const defaultOgImagePath = `https://${baseUrl}/static/og-image.png`
             const ogImagePath = userDefinedOgImagePath ?? generatedOgImagePath ?? defaultOgImagePath
-            const ogImageMimeType = `image/${getFileExtension(ogImagePath) ?? "png"}`
+            const ogImageMimeType = `image/${getFileExtension(ogImagePath)?.slice(1) ?? "png"}`
             return (
               <>
+                <meta property="og:image" content={ogImagePath} />
+                <meta property="og:image:url" content={ogImagePath} />
+                <meta property="og:image:secure_url" content={ogImagePath} />
+                <meta name="twitter:image" content={ogImagePath} />
+                <meta property="og:image:type" content={ogImageMimeType} />
                 {!userDefinedOgImagePath && (
                   <>
                     <meta property="og:image:width" content={fullOptions.width.toString()} />
                     <meta property="og:image:height" content={fullOptions.height.toString()} />
                   </>
                 )}
-
-                <meta property="og:image" content={ogImagePath} />
-                <meta property="og:image:url" content={ogImagePath} />
-                <meta name="twitter:image" content={ogImagePath} />
-                <meta property="og:image:type" content={ogImageMimeType} />
               </>
             )
           },
